docs/application/native/tutorials/process/setting-properties.md

```diff
@@ -137,14 +138,14 @@ You can provide localized versions of the application name (label) and descripti
 
 To add a localized name or description:
 
-- In the **Name** panel, click **+**. Select the language, define the application name for that language, and click **OK**.  
-The following example shows the setting in the `tizen-manifest.xml` file code:  
+- In the **Name** panel, click **+**. Select the language, define the application name for that language, and click **OK**.
```

Comment:
line 141,147,165,166,173,176,179,182: The separated lines are merged as one line.
line 202: Code tag(data type) was not applied all table. 
line 225: Code tag(data type) was not applied all table.

Reply:
line 141,147,165,166,173,176,179,182: fixed.
The other: Please see my previous comments.